gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -46,7 +46,6 @@ properties:
   additional_pathology_findings:
     $ref:
       - "_terms.yaml#/additional_pathology_findings/common"
```

Comment:
We need the enums to be here @ganjigajanan

Reply:
added the enums